.drone.yml

```diff
@@ -121,7 +121,7 @@ steps:
       restore: true
 
   - name: ui-tests
-    image: codedotorg/code-dot-org:1.5.2
+    image: codedotorg/code-dot-org:ruby-2.7.5
```

Comment:
We need to update the signature on this file.

Reply:
See the TODO in the PR description